socorro/unittest/processor/test_breakpad_transform_rules.py

```diff
@@ -310,41 +297,34 @@ def test_stackwalker_fails_2(self, mocked_subprocess_module):
         assert processed_crash.mdsw_status_string == "unknown error"
         assert not processed_crash.success
         expected = [
-            "MDSW output failed in json: Expected String or Unicode",
-            "MDSW failed on 'upload_file_minidump': unknown error"
+            '/bin/stackwalker output failed in json: Expected String or Unicode',
+            (
+                'MDSW failed on \'timeout -s KILL 30 /bin/stackwalker --raw-json '
+                '00000000-0000-0000-0000-000002140504.json --symbols-url https://localhost '
+                '--symbols-url https://localhost --symbols-cache /mnt/socorro/symbols '
+                'a_fake_dump.dump 2>/dev/null\': unknown error'
+            )
         ]
         assert processor_meta.processor_notes == expected
 
     @patch('socorro.processor.breakpad_transform_rules.os.unlink')
     def test_temp_file_context(self, mocked_unlink):
         config = self.get_basic_config()
 
-        rule = BreakpadStackwalkerRule(config)
-        with rule._temp_file_context('foo.TEMPORARY.txt'):
-            pass
-        mocked_unlink.assert_called_once_with('foo.TEMPORARY.txt')
-        mocked_unlink.reset_mock()
-
-        with rule._temp_file_context('foo.txt'):
+        rule = BreakpadStackwalkerRule2015(config)
+        with rule._temp_raw_crash_json_file('foo.json', example_uuid):
```

Comment:
`BreakpadStackwalkerRule` has a `_temp_file_context` context manager which works differently than the `BreakpadStackwalkerRule2015` in a couple of respects:

1. it doesn't check if there's `TEMPORARY` in the filename--it does the same thing for everything
2. it uses a different filename which has the thread name and the crashid in it to reduce thread contention